media-video/cheese/cheese-3.34.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome.org vala virtualx meson
+
+DESCRIPTION="A cheesy program to take pictures and videos from your webcam"
+HOMEPAGE="https://wiki.gnome.org/Apps/Cheese"
+
+LICENSE="GPL-2+"
+SLOT="0/8" # subslot = libcheese soname version
+IUSE="gtk-doc +introspection test X"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+REQUIRED_USE="test? ( X )"
+DEPEND="
```

Comment:
Make this `COMMON_DEPEND` and use it in the definition of `DEPEND` and `RDEPEND` since we're going to have two `DEPEND`-only things.